### PR TITLE
Typo/case fix in #include

### PR DESCRIPTION
**Pull request by dwarfland**
Two meta fixes for the .framework (a typo, and a missing header)

### CODE REVIEW

File: ResearchKit.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -492,7 +492,7 @@
 		BF9155A81BDE8DA9007FA459 /* ORKWaitStep.m in Sources */ = {isa = PBXBuildFile; fileRef = BF9155A21BDE8DA9007FA459 /* ORKWaitStep.m */; };
 		BF9155A91BDE8DA9007FA459 /* ORKWaitStepView.h in Headers */ = {isa = PBXBuildFile; fileRef = BF9155A31BDE8DA9007FA459 /* ORKWaitStepView.h */; };
 		BF9155AA1BDE8DA9007FA459 /* ORKWaitStepView.m in Sources */ = {isa = PBXBuildFile; fileRef = BF9155A41BDE8DA9007FA459 /* ORKWaitStepView.m */; };
-		BF9155AB1BDE8DA9007FA459 /* ORKWaitStepViewController.h in Headers */ = {isa = PBXBuildFile; fileRef = BF9155A51BDE8DA9007FA459 /* ORKWaitStepViewController.h */; settings = {ATTRIBUTES = (Private, ); }; };
+		BF9155AB1BDE8DA9007FA459 /* ORKWaitStepViewController.h in Headers */ = {isa = PBXBuildFile; fileRef = BF9155A51BDE8DA9007FA459 /* ORKWaitStepViewController.h */; settings = {ATTRIBUTES = (Public, ); }; };
```
Comment (umerkhan-apple) [inline ResearchKit.xcodeproj/project.pbxproj]: @dwarfland Agree with Yuan. Let's revert this to `private`.

**Comment by umerkhan-apple** [comment]
r=m3 with one issue.

File: ResearchKit.xcodeproj/project.pbxproj (hunk 1 above)
Comment (dwarfland) [inline ResearchKit.xcodeproj/project.pbxproj]: Well, it's referred to by other public headers?

Comment (YuanZhu-apple) [inline ResearchKit.xcodeproj/project.pbxproj]: Yes, this class should be public.

**Pull request merged**